DEVELOPING.md

```diff
@@ -0,0 +1,27 @@
+# Development Guide
+
+This document contains instructions on how to build and test this image.
+
+## Building the Jetty image
+
+### Local build
+To build the image you need git, docker and maven installed:
+```console
+git clone https://github.com/GoogleCloudPlatform/jetty-runtime.git
+cd jetty-runtime
+mvn clean install
+```
+
+### Cloud build
+To build using the [Google Cloud Container Builder](https://cloud.google.com/container-builder/docs/overview), you need to have git, maven, and the [Google Cloud SDK](https://cloud.google.com/sdk/) installed locally.
+```console
+git clone https://github.com/GoogleCloudPlatform/jetty-runtime.git
+./scripts/cloudbuild.sh
+```
+
+## Running the Jetty image
+The resulting image is called jetty (with more specific tags also created)
+and can be run with:
+```console
+docker run gcr.io/google-appengine/jetty9jetty
```

Comment:
Since this is now inside the DEVELOPING instructions, I think it might make more sense to give the local docker image name here (just `docker run jetty`), since that's what will be produced by a developer's maven build.